core/pkg/hostsensorutils/log_coupling.go

```diff
@@ -0,0 +1,45 @@
+package hostsensorutils
```

Comment:
I noticed all of the functions here are public, but I'm not sure it is needed.
Is there a reason to expose the functions?

Reply:
Not a real reason, actually. I'm going to move them as private.